tests/queries/0_stateless/01888_bloom_filter_hasAny.sql

```diff
@@ -0,0 +1,30 @@
+CREATE TABLE bftest (
```

Comment:
Ok. Let's also add a case with non-constant second argument (index will not work but query should continue to work correctly).

Reply:
Ok, added a test case that shouldn't throw an error, but will just run slowly.